docs/resources/synchronization_job.md

```diff
@@ -0,0 +1,84 @@
+---
+subcategory: "Synchronization"
+---
+
+# Resource: azuread_synchronization_job
+
+Manages a synchronization job associated with an enterprise application service principal within Azure Active Directory. 
+
+## API Permissions
+
+The following API permissions are required in order to use this resource.
+
+When authenticated with a service principal, this resource requires one of the following application roles: `Application.ReadWrite.OwnedBy` or `Directory.ReadWrite.All`
+
+## Example Usage
+
+*Basic example*
+
+```terraform
+data "azuread_application_template" "example" {
+  display_name = "Azure Databricks SCIM Provisioning Connector"
+}
+
+resource "azuread_application" "example" {
+  display_name = "example"
+  template_id  = data.azuread_application_template.example.template_id
+  feature_tags {
+    enterprise = true
+    gallery    = true
+  }
+}
+
+resource "azuread_service_principal" "example" {
+  application_id = azuread_application.example.application_id
+  use_existing   = true
+}
+
+resource "azuread_synchronization_secret" "example" {
+  service_principal_id = azuread_service_principal.example.id
+
+  credentials {
+    key   = "BaseAddress"
+    value = "https://adb-example.azuredatabricks.net/api/2.0/preview/scim"
+  }
+  credentials {
+    key   = "SecretToken"
+    value = "some-token"
+  }
+}
+
+resource "azuread_synchronization_job" "example" {
+  service_principal_id = azuread_service_principal.example.id
+  template_id          = "dataBricks"
+  state                = "Active"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_principal_id` - (Required) The object ID of the service principal for which this synchronization job should be created. Changing this field forces a new resource to be created.
+* `state` - (Optional) Desired state of the provisioning job, 'Active' or 'Disabled'. Default state is Active.
+* `template_id` - (Required) Identifier of the synchronization template this job is based on.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - An ID used to uniquely identify this synchronization job.
+* `schedule` - A `schedule` list as documented below.
+
+---
+
+`schedule` is a list of objects with the following attributes:
+
+* `expiration` - Date and time when this job will expire, formatted as an RFC3339 date string (e.g. `2018-01-01T01:02:03Z`).
+* `interval` - The interval between synchronization iterations ISO8601. E.g. PT40M run every 40 minutes.
+* `state` - State of the job.
+
+## Import
+
+This resource does not support importing.
```

Comment:
We should be able to import this resource?

Reply:
Good point. I added the import functionality.